presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/db/DbResourceGroupConfigurationManager.java

```diff
@@ -157,7 +163,16 @@ public synchronized void load()
             this.resourceGroupSpecs = resourceGroupSpecs;
             this.cpuQuotaPeriod.set(managerSpec.getCpuQuotaPeriod());
             this.rootGroups.set(managerSpec.getRootGroups());
-            this.selectors.set(buildSelectors(managerSpec));
+            List<ResourceGroupSelector> selectors = buildSelectors(managerSpec);
+            if (enableExactMatchSelector) {
```

Comment:
I think you can move this `if` logic into the `buildSelectors` method.

Reply:
I thought about that, but that's in an abstract class, and not all of them support this. We can make this a required thing for all classes extends that but that is not absolutely necessary. I'm open for discussion. But consider the urgency, I'd say if we want to do the refactoring, let's do it later?

Reply:
I see, then it makes sense to keep it here, otherwise it will be more confusing as json stuff doesn't even support this.